src/components/Metatags/Metatags.tsx

```diff
@@ -0,0 +1,187 @@
+import { IDocsEntity } from '@api';
+import React from 'react';
+import Head from 'next/head';
+import { Esources } from '@api/lib/search/types';
+
+const ADS_BASE_URL = 'https://ui.adsabs.harvard.edu';
+
+const ABS_BASE_URL = `${ADS_BASE_URL}/abs`;
+
+const LINKGWAY_BASE_URL = `${ADS_BASE_URL}/link_gateway`;
+interface IMetatagsProps {
+  doc: IDocsEntity;
+}
+
+export const metatagsQueryFields: Partial<keyof IDocsEntity>[] = [
+  'abstract',
+  'aff',
+  'author',
+  'bibcode',
+  'bibstem',
+  'doctype',
+  'doi',
+  'esources',
+  'identifier',
+  'isbn',
+  'issn',
+  'issue',
+  'keyword',
+  'page',
+  'page_range',
+  'pub',
+  'pubdate',
+  'title',
+  'volume',
+];
+
+export const Metatags = (props: IMetatagsProps): React.ReactElement => {
+  const { doc } = props;
+
+  if (!doc) {
+    return null;
+  }
+
+  const getFomattedNumericPubdate = (pubdate: string) => {
+    const regex = /(....)-(..)-(..)/;
+    const res = regex.exec(pubdate);
+    return `${res[1]}/${res[2]}`;
```

Comment:
I think we should change this up a little bit, one you could make this regex more explicit somewhat by doing something like:
`/^(\d{4})-(\d{2})-(\d{2})$/`
But I think it would be better to use the native Date for this:

```
import { format } from 'date-fns';

try {
  return format(new Date(pubdate), 'yyyy/MM');
} catch (e) {
  // date breaks, return an empty string
  return '';
}
```